commit ea07de92da8f51c0c1b78a10f197ad6ab1a39aa0
Author: Rodrigo Vivi <rodrigo.vivi@intel.com>
Date:   Tue Mar 1 17:07:04 2016 -0800

    intel: Adding missing Broxton PCI IDs.
    
    These IDs were already part of the kernel since:
    
    kernel commit 985dd4360fdf2533fe48a33a4a2094f2e4718dc0
    Author: Imre Deak <imre.deak@intel.com>
    Date:   Thu Jan 28 16:04:12 2016 +0200
    
        drm/i915/bxt: update list of PCIIDs
    
    Cc: Venkateswarlu Vinjamuri <venkateswarlu.v.vinjamuri@intel.com>
    Signed-off-by: Rodrigo Vivi <rodrigo.vivi@intel.com>
    Reviewed-by: Clint Taylor <Clinton.A.Taylor@intel.com>

diff --git a/intel/intel_chipset.h b/intel/intel_chipset.h
index 35148e5..8e42a40 100644
--- a/intel/intel_chipset.h
+++ b/intel/intel_chipset.h
@@ -213,6 +213,8 @@
 #define PCI_CHIP_BROXTON_0		0x0A84
 #define PCI_CHIP_BROXTON_1		0x1A84
 #define PCI_CHIP_BROXTON_2		0x5A84
+#define PCI_CHIP_BROXTON_3		0x1A85
+#define PCI_CHIP_BROXTON_4		0x5A85
 
 #define IS_MOBILE(devid)	((devid) == PCI_CHIP_I855_GM || \
 				 (devid) == PCI_CHIP_I915_GM || \
@@ -436,7 +438,9 @@
 
 #define IS_BROXTON(devid)	((devid) == PCI_CHIP_BROXTON_0	|| \
 				 (devid) == PCI_CHIP_BROXTON_1	|| \
-				 (devid) == PCI_CHIP_BROXTON_2)
+				 (devid) == PCI_CHIP_BROXTON_2	|| \
+				 (devid) == PCI_CHIP_BROXTON_3	|| \
+				 (devid) == PCI_CHIP_BROXTON_4)
 
 #define IS_GEN9(devid)		(IS_SKYLAKE(devid) || \
 				 IS_BROXTON(devid) || \
